src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -147,4 +148,13 @@ private boolean companiesAreUnique(List<Company> companies) {
         }
         return true;
     }
+
+    /**
+     * Sorts the list in alphabetical order.
+     * Sort is non case sensitive.
+     */
+    public void sort() {
+        Comparator<Company> compareByName = Comparator.comparing(company -> company.getName().fullName.toLowerCase());
+        FXCollections.sort(internalList, compareByName);
```

Comment:
Do you think it would be better to add a few test cases for this?

Reply:
Added the test cases